### PR TITLE
FOGL-941 - Updated doc

### PR DESCRIPTION
**Pull request by izoratti**
@MarkRiddoch - Looking at you doc and the output from the rest API, things are slightly different - could you pls have a look? 
Specifically:
05_testing.html#the-omf-translator >>> $ curl -s http://localhost:8081/foglamp/schedule
vs.
https://docs.google.com/document/d/1JJDP7g25SWerNVCxgff02qp9msHbqA9nt3RAFx8-Qng/edit#heading=h.daqvxnnb7r5s

@sstefano69 - pls have a look at the same page for the example regarding SEND_PR_1.

### CODE REVIEW

File: docs/05_testing.rst [hunk 1]
```diff
@@ -439,99 +447,190 @@ If you are curious to see which categories are available in FogLAMP, simply type
...
+                           "type": "integer", "value": "5000", "default": "5000" },
+    "applyFilter":       { "description": "Whether to apply filter before processing the data",
+                           "type": "boolean", "value": "False", "default": "False" },
+    "stream_id":         { "description": "Stream ID",
+                           "type": "integer", "value": "1", "default": "1" }
   }
   $
 
-What we need to do now is to change the IP address in the URL of *SEND_PR_1* with the IP address of the Windows Server running the PI Connector Relay. Supposing the Windows IP address is *192.168.56.101*, we can use curl for this:
+You can look at the descriptions to have a taste of what you can control with this plugin. The configuration should be fine, with the exception of the *URL*, which of course should refer to the IP address of the machine and the port used by the PI Connector Relay OMF. Assuming, for example, that the port is *5460* and the IP address is *192.168.56.101*, you can set the new URL with this PUT method:
```
Comment (MarkRiddoch) [inline docs/05_testing.rst]: The default configuration should be fine

File: docs/07_admin/02_utilities.rst [hunk 2]
```diff
@@ -52,7 +59,15 @@ In more specific terms, the command executes these steps:
...
-``foglamp stop`` is the command used to stop FogLAMP. In its current implementation, the command kills the processes associated to the FogLAMP microservices on the machine. A later version will execute a more graceful shutdown.
+``foglamp stop`` is the command used to stop FogLAMP. The command waits until all the tasks and services have been completed, then it stops the core service.
+
+
+If Foglamp Does Not Stop
+------------------------
+
+If FogLAMP does not stop, i.e. if by using the process status command ``ps`` you see FogLAMP processes still running, you can use ``foglamp kill`` to kill them.
+
+.. note:: The command issues a ``kill -9`` against the processes associated to FogLAMP. This is not recommended, unless FogLAMP cannot be stopped. th the *stop* command. In other words, *kill* is your last resort before a reboot. If you must use the kill command, it means that there is a problem: let us know about that by submitting an issue on |GH Issues|.
```
Comment (MarkRiddoch) [inline docs/07_admin/02_utilities.rst]: th the

File: docs/05_testing.rst [hunk 3]
```diff
@@ -439,99 +447,190 @@ If you are curious to see which categories are available in FogLAMP, simply type
...
+    "type":        "string",
+    "value":       "https://192.168.56.101:5460/ingress/messages",
+    "default":     "https://pi-server:5460/ingress/messages"
+  }
   $
 
-By using the PUT method, FogLAMP replies with the new value of the entry. You can note that the *value* element is the only one that can be changed in *URL* (the other elements are factory settings).
+You can note that the *value* element is the only one that can be changed in *URL* (the other elements are factory settings).
 
 Now we are ready to send data North, to the PI System.
```
Comment (sstefano69) [inline docs/05_testing.rst]: Note : the command line prompt showing debug messages is no more available using the version 1.2

**Comment by sstefano69** [comment]
text :  *OMF Translator* is schedules as a North task...
to :     *OMF Translator* is scheduled as a North task...

**Comment by sstefano69** [comment]
text : (the time may vary, we set it to 30 seconds to facilitate the testing.
to : (the time may vary, we set it to 30 seconds to facilitate the testing).

File: docs/05_testing.rst [hunk 4]
```diff
@@ -439,99 +447,190 @@ If you are curious to see which categories are available in FogLAMP, simply type
...
+You can look at the descriptions to have a taste of what you can control with this plugin. The configuration should be fine, with the exception of the *URL*, which of course should refer to the IP address of the machine and the port used by the PI Connector Relay OMF. Assuming, for example, that the port is *5460* and the IP address is *192.168.56.101*, you can set the new URL with this PUT method:
 
 .. code-block:: console
 
-  $ curl -H 'Content-Type: application/json' -X PUT -d '{ "value": "http://192.168.56.101:8118/ingress/messages" }' http://localhost:8081/foglamp/category/SEND_PR_1/URL;echo
-  {"value": "http://192.168.56.101:8118/ingress/messages", "default": "http://WIN-4M7ODKB0RH2:8118/ingress/messages", "description": "The URL of the PI Connector to send data to", "type": "string"}
+  $ curl -sH'Content-Type: application/json' -X PUT -d '{ "value": "https://192.168.56.101:5460/ingress/messages" }' http://localhost:8081/foglamp/category/SEND_PR_1/URL | jq
```
Comment (sstefano69) [inline docs/05_testing.rst]: we maybe highlight that the port 5460 (and https) is for the Connect Relay 1.2 (the previous one was using 8118 and http)

**Pull request merged**